src/components/Verify.tsx

```diff
@@ -48,6 +72,7 @@ export const Verify = () => {
   });
   const onSubmit = handleSubmit(async (values) => {
     console.log('formData', values);
+    setForm(values);
```

Comment:
I think the problem with this setup right now is that we want to make the api call within `handleSubmit` right? So rather than return only data we will need to return the fetch function to call. 

A way to simplify some of this may be to wrap the `useMutation` hook in swr.

https://swr.vercel.app/docs/mutation

Reply:
Check out d82de36 to see what I did, but ended up not using swr because, since we don't POST on load and wait for form submit, it didn't really add any value

Reply:
Hmm I haven't yet been able to get the error handling as nice with this approach. Use the default data and change the last digit of the address from `2` to `3` and try verifying. With the current approach, below the verify button it should say "No deployed code found for contract"

But when I try the swr approach I either get a runtime error (when that react failure block modal pops up and blocks the screen) or a string of "Unexpected token 'N', "No deploye"... is not valid JSON" because it tries to parse the actual error as json, depending on what I try.

If you want to open a PR I'm happy to merge it as long as it keeps the error handling the same